adapters/richaudience/richaudiencetest/exemplary/single-banner-deviceConfig.json

```diff
@@ -0,0 +1,127 @@
+{
+    "mockBidRequest": {
+      "id": "4d3f84eb-787b-42fb-a0cf-062690dadce3",
+      "test": 0,
+      "imp": [
+        {
+          "id": "div-gpt-ad-1460505748561-0",
+          "banner": {
+              "format" : [{
+                    "w": 300,
+                    "h": 250
+                  }]
+          },
+          "bidfloor": 1e-05,
+          "bidfloorcur": "USD",
+          "ext": {
+           "bidder": {
+              "pid": "OsNsyeF68q",
+              "supplyType": "site",
+              "test": true
+            }
+          }
+        }
+      ],
+      "site": {
+        "domain": "bridge.richmediastudio.com",
+        "page": "https://bridge.richmediastudio.com//ab083674fb8200b877a6983126e4477d/PrebidServer/indexRa.html?pbjs_debug=true"
+      },
+      "device": {
+          "dnt": 1,
+          "lmt": 1,
+        "ip": "11.222.33.44"
+    },
+      "user": {
+        "buyeruid": "189f4055-78a3-46eb-b7fd-0915a1a43bd2a",
+        "ext": {}
+      }
+    },
+    
+    "httpCalls": [
+      {
+        "expectedRequest": {
+          "uri": "http://ortb.richaudience.com/ortb/?bidder=pbs",
+          "body": {
```

Comment:
Should you specify the headers here as you have in other test files? If the headers are missing, the check will be skipped.